packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -285,8 +296,21 @@ export default {
                 : [this.loadAvailableFulfilment()];
 
             await Promise.all(promises);
+
+            if (this.shouldLoadAddress()) {
+                await this.loadAddress();
+            }
         },
 
+        /**
+         * Check if address should be loaded from a getAddress API endpoint
+         *
+         */
+        shouldLoadAddress () {
```

Comment:
This should probably be a computed property

Reply:
@xander-marjoram fixed